### PR TITLE
[WIP] A bunch of fixes and additions

### PR DESCRIPTION
**Pull request by SoSeDiK**
Updated version to 1.7.0, updated dependencies.
Added a new SF machine (Kitchen) for CustomFood from ExoticGarden and a research for it.
Denied consuming skulls when clicked on the furnace, workbench, chest etc.
Renamed "Apple" to "Oak Apple" to avoid confusion in future.
Allowed making an Apple Juice from an Apple Oak.
Added "Dragon Fruit" _(Finally! Hooray!)_ and basic recipes for it.
Changed CustomPotion, so now you can set a custom potion color.
Replaced all § with &.
Replaced all deprecated methods, where it was possible.
Moved schematics to their own folder (/plugins/ExoticGarden/schematics/).
Fixes for the issues #17, #19, #21, #27.
Solves #14 _(will work until you install another plugin that uses the "me.mrCookieSlime.ExoticGarden.Schematic.org.jnbt" package)_, #24 _(EG will now listen for all region protection plugins that are supported in CS-CoreLib)_ and #20 _(I have this issue only with ORANGE_TREE, so it was fixed only for it)_.

### CODE REVIEW

File: src/me/mrCookieSlime/ExoticGarden/Kitchen.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+								for (int i = 0; i < inputs.size(); i++) {
+									boolean craft = true;
+									for (int j = 0; j < inv.getContents().length; j++) {
+										if (!SlimefunManager.isItemSimiliar(inv.getContents()[j], inputs.get(i)[j], true)) {
+											craft = false;
+											break;
+										}
+									}
+									if (craft) {
```
Comment (xH3LLRAIZ3Rx) [inline src/me/mrCookieSlime/ExoticGarden/Kitchen.java]: I tried to craft something in it (a jelly sandwich) and it says the recipe cannot be found.

Comment (SoSeDiK) [reply src/me/mrCookieSlime/ExoticGarden/Kitchen.java]: Thank you. It should work now :)

**Comment by SoSeDiK** [comment]
Tested with Bukkit 1.12.2. Should work fine with other versions too.
I'll be thankful if someone will test it on an actual server with players.

**Comment by galacticwarrior9** [comment]
Works fine! The update is much appreciated and has not gone unnoticed.

**Comment by xH3LLRAIZ3Rx** [comment]
you should update to 1.13

**Comment by SoSeDiK** [comment]
The problem is that I have no computer right now. And I'll be busy with studying for the next few months. I'll update this branch to 1.13 in my holidays /// and maybe upload my local Slimefun and CS-CoreLib forks with 1.13 support if anyone else won't, but I dropped support for everything older than 1.13, so my changes probably won't be merged.

**Comment by TheBusyBiscuit** [comment]
Merged and compiled successfully.
https://thebusybiscuit.github.io/builds/TheBusyBiscuit/ExoticGarden/master/

**Pull request merged**